docs/migration/migrate_7_0.asciidoc

```diff
@@ -9,6 +9,13 @@ your application to Kibana 7.0.
 
 See also <<release-highlights>> and <<release-notes>>.
 
+[float]
+=== Removed support for user's relying on direct index privileges to the Kibana index in Elasticsearch
```

Comment:
Sorry, missed one:
```suggestion
=== Removed support for users relying on direct index privileges to the Kibana index in Elasticsearch
```

Reply:
Thanks... I don't know what happened to my grammar/english skills.